### ISSUE TITLE
Business Insider Adblock detection

### ISSUE THREAD

**Issue by xtrah69** [reek/anti-adblock-killer#3187]
Adguard Desktop

English, Spyware, Social media, annoyances and AAK

Article: http://uk.businessinsider.com/easiest-universities-to-get-into-in-the-uk-2017-5

Screenshot: http://prntscr.com/f6al6n

Thanks!

**Comment by uBlock-user** [comment]
Remove AAK and install Adblock Protector Script, it's patched there.

**Comment by xtrah69** [comment]
Thank you!

**Comment by xtrah69** [comment]
I've installed Adblock Protector Script, still seems to be there! It's been installed properly in Adguard Desktop as both userscript and filter are active.

**Comment by uBlock-user** [comment]
It shouldn't. It's patched in the script. Adguard for userscript is not supported, so you will need to install TamperMonkey in your browser.

**Comment by xtrah69** [comment]
I'll give that a whirl, thanks!

**Comment by xtrah69** [comment]
Will Adguard for userscript be supported anytime soon?

**Comment by xtrah69** [comment]
Yeah, it still happens, Adguard has the filter active and Tampermonkey is in use.
![image](https://cloud.githubusercontent.com/assets/24882469/25972192/e3e88f48-3696-11e7-8e59-d1a1b649582f.png)

**Comment by xtrah69** [comment]
![image](https://cloud.githubusercontent.com/assets/24882469/25972206/f29065c0-3696-11e7-8b00-2ee6baab51bb.png)

**Comment by uBlock-user** [comment]
Can't reproduce. Works fine on my end.

**Comment by xtrah69** [comment]
Probably because I'm not using uBlock Origin I guess. Thanks anyway! I'll open up issues on Adblock Protector Script from now on seeing as I've abandoned AAK!

**Comment by mbierman** [comment]
http://www.businessinsider.com/elon-musk-turtlenecks-steve-jobs-2017-11?nr_email_referer=1&utm_source=Sailthru&utm_medium=email&utm_content=TechSelect&pt=385758&ct=Sailthru_BI_Newsletters&mt=8&utm_campaign=BI%20Tech%20%28Wednesday%20Friday%29%202017-11-15&utm_term=Tech%20Select%20-%20Engaged%2C%20Active%2C%20Passive%2C%20Disengaged
Google Chrome Version 62.0.3202.94 (Official Build) (64-bit)
TamperMonkey4.4
Adblock Plus  1.13.4
x   Enabled Adblock Warning Removal List Last updated at 9:55:39 AM today
×   Enabled EasyList Last updated at 1:51:13 PM on 11/14/2017
×   Enabled AakList (Anti-Adblock Killer) Last updated at 9:55:39 AM today

![image](https://user-images.githubusercontent.com/1205471/32853276-9d1fc0c6-c9ef-11e7-9ddc-d17657e920fb.png)
